src/ballet/chacha20/fd_chacha20.h

```diff
@@ -0,0 +1,14 @@
+#ifndef HEADER_fd_src_ballet_chacha20_fd_chacha20_h
+#define HEADER_fd_src_ballet_chacha20_fd_chacha20_h
+
+#include <openssl/evp.h>
+
+typedef unsigned int fd_chacha20_rng_t;
+
+#define FD_CHACHA20_KEY_SIZE 32   // 32 bytes, 256 bits
+#define FD_CHACHA20_NONCE_SIZE 12 // 12 bytes, 96 bits
+
+int fd_chacha20_rng_init(unsigned char *key, unsigned char *nonce);
+int fd_chacha20_rng_get_uint32(fd_chacha20_rng_t *num);
```

Comment:
These functions in the header need some documentation. See other header files in the project to get a sense of what I mean.

Reply:
Just  updated this by deleting these methods from header (there are now internal functions). Now we're exposing a single method to generate random numbers

Reply:
Definitely need to add some comments to document this as well